token/core/zkatdlog/crypto/validator/validator.go

```diff
@@ -103,6 +179,21 @@ func (v *Validator) VerifyTokenRequest(ledger driver.Ledger, signatureProvider d
 		actions = append(actions, action)
 	}
 
+	for _, sig := range signatureProvider.Signatures() {
```

Comment:
why is this not performed by the extravalidator?

Reply:
the extravalidator is there so it would be easier to introduce different types of scripts 
since this applies only to the exchange script no reason to have an extravalidator for it